aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -187,6 +190,23 @@ module aptos_framework::account {
     /// Create signer for testing, independently of an Aptos-style `Account`.
     public fun create_signer_for_test(addr: address): signer { create_signer(addr) }
 
+    struct AccountPermission has copy, drop, store {}
```

Comment:
I don't follow fully what all of the function below do - but not all are related to key rotation, right?

If so, can we organize them, and have AccountKeyRotationPermission as a separate category, and see for the rest if we need 1 or more.

If all below are related to key rotation - then rename to AccountKeyRotationPermission - as that sounds much scarier (as it should), than AccountPermission